docs/overview/index.md

```diff
@@ -3,8 +3,52 @@ id: overview_index
 title: "Contents"
 ---
 
-TODO: List of contents available
-
 ## Installation
 
-TODO: Installation details
+Add in your `build.sbt`:
+
+```scala
+libraryDependencies += "dev.zio" %% "zio-schema" % "<version>"
+```
+
+## Purpose of ZIO-Schema
+ZIO-Schema allows you to create representations of your data types as values. 
+
+Once you have a representation of your data types, you can use it to 
+ - serialize and deserialize your types
+ - validate your types
+ - transform your types
+ - create instances of your types
+
+You can then use one of the various codecs (or create your own) to serialize and deserialize your types.
+
+Example of codecs are:
```

Comment:
Here I would specify which codecs are already implemented vs which ones are possible. Currently we have protobuf and json codecs provided (with Thrift and Avro on the way).